models/inventory/tests/testStockTransfer.spec.ts

```diff
@@ -12,6 +12,11 @@ import { Shipment } from '../Shipment';
 import { StockTransfer } from '../StockTransfer';
```

Comment:
Following tests are missing:
- Creating Payments for Credit Note
- Creating Payments for Debit Note
- Creating Credit Note if Invoice is not paid
- Creating Debit Note if Invoice is not paid